### ISSUE TITLE
Infinite loop

### ISSUE THREAD

**Issue by farshid-vahid** [CredibilityLab/groundhog#101]
```
---------------------------------------------------------------------------
|   IMPORTANT.
|       The path 'C:/Users/farshid/Dropbox/R_groundhog/groundhog_library/' 
|       seems to be a Dropbox folder. Groundhog will be slower and more 
|       likely to produce occasional errors if the library is on Dropbox; 
|       consider using a different path. If you want to set it to  
|       'C:/Users/farshid/Dropbox/R_groundhog/groundhog_library/' anyway, 
|       type 'anyway', else type 'cancel' 
|   >cancel
OK. Request cancelled.

---------------------------------------------------------------------------
|   IMPORTANT.
|       The path 'C:/Users/farshid/Dropbox/R_groundhog/groundhog_library/' 
|       seems to be a Dropbox folder. Groundhog will be slower and more 
|       likely to produce occasional errors if the library is on Dropbox; 
|       consider using a different path. If you want to set it to  
|       'C:/Users/farshid/Dropbox/R_groundhog/groundhog_library/' anyway, 
|       type 'anyway', else type 'cancel' 
|   >
```
I typed cancel many many times, but, even though it says "OK. Request cancelled", the loop repeats. I had to close down R. I am using R 4.3.1

**Comment by urisohn** [comment]
I was not able to reproduce this. In R 4.3.1 in Windows. When I type 'cancel' it stopped without a loop.
I even created the folder in Dropbox and still it would not produce a loop.

From the path I assume you are also using Windows so a bit puzzling.

1. Are you using groundhog 3.1.0?
2. Have you used groundhog before and this is a new bug for you, or is this your first time using it?
3. Is there any other code you ran before that?
4. If you run `get.groundhog.folder()`, what do you get?

If you are in the loop again, you can restart the R sessino within R to get out of there, without closing it down.
SHFT-CTRL-F10

**Comment by farshid-vahid** [comment]
Thanks Uri. Yes, I am using Windows.
1. Yes, I was using groundhog 3.1.0
2. That was the first time using it
3. No, I just read about groundhog, and installed groundhog and ran this script: 
``` 
library(groundhog)
today <- "2023-08-08"
groundhog.library(xts,today) 
```
Anyway, I found out that my environment variable R_USER was set to a dropbox folder, so I changed that, and then opened R and everything is fine. Now, when I run `get.groundhog.folder()` I get my "R_USER path/R_groundhog/groundhog_library/".
